docs/deploy-options.md

```diff
@@ -82,25 +75,25 @@ Then run the create cluster command passing the config file as a parameter.
 This file is in the `examples/kind` directory:
 
 ```bash
-$ kind create cluster --config examples/kind/kind-expose-port.yaml
+kind create cluster --config examples/kind/kind-expose-port.yaml
```

Comment:
Please put the $ back in all the examples that it was remove from -- or remove it from every other example. I think the former is simpler

Reply:
My VSCode has a linter that was complaining about $ signs with no output, I'll just ignore it.

Reply:
I don't have a strong position on should there be a shell sigil, just that its inconsistent to change some, but not all. If you want to go and delete them all so vscode smiles upon your countenance, go for it.